core/src/test/java/greencity/validator/LanguageValidatorTest.java

```diff
@@ -0,0 +1,45 @@
+package greencity.validator;
+
+import greencity.service.LanguageService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.util.Locale;
+
+import static greencity.repository.ModelUtils.getAllLanguages;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.when;
+
+
+@ExtendWith(MockitoExtension.class)
+public class LanguageValidatorTest {
```

Comment:
SonarLint point to this public identifier
JUnit5 test classes and methods should have default package visibility

Reply:
Fixed. Thanks for review!